### PR TITLE
Jersey upgrade

### PR DESCRIPTION
**Pull request by thongsav-usgs**

### CODE REVIEW

**Comment by thongsav-usgs** [comment]
@jiwalker-usgs, this is actually "good enough" to merge now. I haven't done the full blown MVC refactor yet, but all pages seem to work now.

**Comment by jiwalker-usgs** [comment]
So I should try to resolve the conflicts then?

**Comment by thongsav-usgs** [comment]
Yes, please. The sooner we get everyone on hte same page the better. I am working on securing end points now.

File: coastal-hazards-portal/src/main/webapp/js/cch/objects/publish/UI.js [hunk 1]
```diff
@@ -1641,6 +1642,7 @@ CCH.Objects.Publish.UI = function () {
 							wfsServiceVal = baseService + 'wfs',
 							wmsServiceVal = baseService + 'wms',
 							updateAttributesCallback;
+					debugger
```
Comment (jiwalker-usgs) [inline coastal-hazards-portal/src/main/webapp/js/cch/objects/publish/UI.js]: Probably don't want to check this in in the future.

File: coastal-hazards-portal/src/main/webapp/js/cch/objects/publish/UI.js [hunk 2]
```diff
@@ -166,6 +166,7 @@ CCH.Objects.Publish.UI = function () {
 
 	me.enableNewItemForm = function () {
 		var gsBaseUrl = CCH.CONFIG.contextPath + CCH.CONFIG.data.sources[CCH.CONFIG.strings.cidaGeoserver].proxy + 'proxied/';
+		debugger
```
Comment (jiwalker-usgs) [inline coastal-hazards-portal/src/main/webapp/js/cch/objects/publish/UI.js]: this too

Comment (isuftin) [inline coastal-hazards-portal/src/main/webapp/js/cch/objects/publish/UI.js]: This project won't build if this is in here, actually. yuicompressor-maven-plugin will break the build

Comment (thongsav-usgs) [inline coastal-hazards-portal/src/main/webapp/js/cch/objects/publish/UI.js]: Totally my bad. Will fix.

Comment (jiwalker-usgs) [inline coastal-hazards-portal/src/main/webapp/js/cch/objects/publish/UI.js]: no worries, I fixed it in the other PR.

**Comment by thongsav-usgs** [comment]
Closing this, will take in changes from: https://github.com/USGS-CIDA/coastal-hazards/pull/994

**Pull request closed without merging**